app/scripts/templates/delete_account.mustache

```diff
@@ -1,38 +1,29 @@
-<div id="main-content" class="card">
-  <header>
-    <h1 id="fxa-delete-account-header">{{#t}}Delete account{{/t}}</h1>
-  </header>
+<div id="delete-account" class="settings-unit">
+  <div class="error"></div>
 
-  <section>
-    <div class="success"></div>
-    <div class="error"></div>
+  <div class="settings-unit-stub">
+    <header class="settings-unit-summary">
+      <h2 class="settings-unit-title">{{#t}}Delete account{{/t}}</h2>
+    </header>
+    <button class="settings-button secondary settings-unit-toggle hidden">{{#t}}Delete {{/t}}</button>
+  </div>
 
+  <div class="settings-unit-details">
     <form novalidate>
       <p>{{#t}}Are you sure you want to delete your account? This action cannot be undone.{{/t}}</p>
-      <p class="prefill">{{ email }}</p>
 
       <div class="input-row password-row">
-        <!-- add the autocomplete=false attribute - the user is
-             deleting the account, they most likely don't want
-             the password saved. -->
+        <!-- add the autocomplete=false attribute - the user is deleting the account, they most likely don't want the password saved. -->
```

Comment:
`autocomplete=false` or `autocomplete="off"`? The comment doesn't seem to match the code on L17 below.